lib/fakes/pacman_service.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+class Fakes::PacmanService < ExternalApi::PacmanService
+  class << self
+    def send_communication_package_request(file_number, name, document_references)
+      fake_package_request(file_number, name, document_references)
+    end
+
+    def send_distribution_request(package_id, recipient, destinations)
+      fake_distribution_request(package_id, recipient, destinations)
+    end
+
+    def get_distribution_request(distribution_id)
+      if distribution_id != "673c8b4a-cb7d-4fdf-bc4d-998d6d5d7431"
+        return distribution_not_found_response
+      end
+
+      fake_distribution_response(distribution_id)
+    end
+
+    private
+
+    def bad_request_response
+      HTTPI::Response.new(
+        400,
+        {},
+        OpenStruct.new(
+          "error": "BadRequestError",
+          "message": "participant id is not valid"
+        )
+      )
+    end
+
+    def bad_access_response
+      HTTPI::Response.new(
+        403,
+        {},
+        OpenStruct.new(
+          "error": "BadRequestError",
+          "message": "package cannot be created because of insufficient privileges"
+        )
+      )
+    end
+
+    def distribution_not_found_response
+      HTTPI::Response.new(
+        404,
+        {},
+        OpenStruct.new(
+          "error": "BadRequestError",
+          "message": "distribution does not exist at this time"
+        )
+      )
+    end
+
+    # POST: /package-manager-service/communication-package
+    def fake_package_request(file_number, name, document_references)
+      HTTPI::Response.new(
+        201,
+        {},
+        OpenStruct.new(
+          "id": "24eb6a66-3833-4de6-bea4-4b614e55d5ac",
+          "fileNumber": file_number,
+          "name": name,
+          "documentReferences": document_references,
+          "status": "NEW",
+          "createDate": ""
+        )
+      )
+    end
+
+    # rubocop:disable Metrics/MethodLength
+    # POST: /package-manager-service/distribution
+    def fake_distribution_request(package_id, recipient, destinations)
+      HTTPI::Response.new(
+        201,
+        {},
+        OpenStruct.new(
+          "id": "12345",
+          "recipient": recipient,
+          "description": "bad",
+          "communicationPackageId": package_id,
+          "destinations": destinations,
+          "status": "",
+          "sentToCbcmDate": ""
+        )
+      )
+    end
+
+    # GET: /package-manager-service/distribution/{id}
+    def fake_distribution_response(distribution_id)
+      HTTPI::Response.new(
+        200,
+        {},
+        OpenStruct.new(
+          "id": "673c8b4a-cb7d-4fdf-bc4d-998d6d5d7431",
```

Comment:
```suggestion
    def fake_distribution_response(distribution_id)
      HTTPI::Response.new(
        200,
        {},
        OpenStruct.new(
          "id": distribution_id,
```
I think this UUID will be whichever `distribution_id` is passed in.